README.md

```diff
@@ -380,6 +378,24 @@ add-zsh-hook chpwd load-nvmrc
 load-nvmrc
 ```
 
+#### bash
+
+Put this into your `$HOME/.bashrc` to call `nvm use` automatically whenever you enter a directory that contains an
+`.nvmrc` file with a string telling nvm which node to use:
+
+```bash
+# place this after nvm initialization!
+cd() {
+	if [ -z "$1" ]
+	then
+		builtin cd
```

Comment:
Some shells error when `cd` is given multiple arguments (https://unix.stackexchange.com/questions/373625/different-behaviour-of-cd-with-multiple-arguments-in-bash-releases) - it'd be good to preserve that.